src/quacc/recipes/newtonnet/core.py

```diff
@@ -0,0 +1,416 @@
+"""
+Core recipes for the NewtonNet code
+"""
+from __future__ import annotations
+
+from typing import Dict, Any, Literal
+from copy import deepcopy
+
+import numpy as np
+from ase.atoms import Atoms
+from ase.optimize.optimize import Optimizer
+from ase.vibrations.data import VibrationsData
+from monty.dev import requires
+
+import covalent as ct
+from quacc import SETTINGS
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_calc
+from quacc.util.thermo import ideal_gas
+
+try:
+    from sella import IRC, Sella
+except ImportError:
+    Sella = None
+
+try:
+    from newtonnet.utils.ase_interface import MLAseCalculator as NewtonNet
+except ImportError:
+    NewtonNet = None
+
+
+def _get_hessian(atoms: Atoms) -> np.ndarray:
+    """
+    Calculate and retrieve the Hessian matrix for the given molecular configuration.
+
+    This function takes an ASE Atoms object representing a molecular configuration and uses the
+    NewtonNet machine learning calculator to calculate the Hessian matrix. The calculated Hessian
+    matrix is then returned.
+
+    Args:
```

Comment:
Can you format the docstrings to be numpy format (like the others)? See the `-------` separator, "Parameters" instead of "args", etc.

This will ensure that the docstring renders correctly in the documentation. Thanks!